test/JDBC/input/functions/sys-host_name-vu-prepare.sql

```diff
@@ -0,0 +1,42 @@
+CREATE VIEW host_name_view AS SELECT DISTINCT hostname FROM sys.sysprocesses WHERE spid = @@SPID
```

Comment:
please use filename_xxx as object name. Also, for release where input prepare file can be used, add it to schedule file instead of creating a copy in release directory. If we need changes for specific release, copy is fine.

Reply:
Actually if any one of the prepare, verify and cleanup files' output is different across releases, then we have to add all the three files in specific release folder. So in my case, prepare file is common as you mentioned. If I add an entry in `13_6/schedule` for `sys-host_name`, the framework will assume that file `sys-host_name-vu-verify.sql` and its output will match with the common expected output i.e. framework will use output file `sys-host_name-vu-verify.out` instead of using a version specific expected output `14_latest__verification_cleanup__13_6__sys-host_name-vu-verify.out`.

Reply:
that is fine, if we have difference in behaviuor/output across releases, we can add a copy